### PR TITLE
Expose Avro class in ETL lib batch jar

### PR DESCRIPTION
**Pull request by gokulavasan**
Tested on cluster (throws ClassNotFoundException: Class org.apache.avro.mapreduce.AvroKeyOutputFormat without this fix).

Bamboo : http://builds.cask.co/browse/CDAP-RT22-1

### CODE REVIEW

File: cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml [hunk 1]
```diff
@@ -198,7 +198,7 @@
               <instructions>
                 <!-- build bundle jar for batch. excluding realtime-->
                 <Export-Package>co.cask.cdap.templates.etl.batch.*;co.cask.cdap.templates.etl.transforms.*,
-                  co.cask.cdap.templates.etl.common</Export-Package>
+                  co.cask.cdap.templates.etl.common,org.apache.avro.mapreduce</Export-Package>
```
Comment (chtyim) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/pom.xml]: either fix to ";" or "," as separator.

**Comment by gokulavasan** [comment]
@chtyim Addressed comment. Please review.

**Comment by rohitsinha54** [comment]
Doesn't this need avro-mapred too ?

**Comment by chtyim** [comment]
LGTM

**Comment by chtyim** [comment]
@rohitsinha54 You mean avro-mapred as a dependency? I think the etl-lib module is already depending on it.

**Pull request merged**